netci.groovy

```diff
@@ -1711,15 +1711,20 @@ combinedScenarios.each { scenario ->
                                 case 'x86ryujit':
                                 case 'x86lb':
                                     def arch = architecture
-                                    if (architecture == 'x86ryujit' || architecture == 'x86lb') {
+                                    def buildOpts = ''
+                                    if (architecture == 'x86ryujit') {
+                                        arch = 'x86'
+                                        buildOpts = 'altjitcrossgen'
+                                    }
+                                    else if (architecture == 'x86lb') {
                                         arch = 'x86'
                                     }
                                     
                                     if (Constants.jitStressModeScenarios.containsKey(scenario) ||
                                             scenario == 'default' ||
                                             scenario == 'r2r' ||
                                             Constants.r2rJitStressScenarios.indexOf(scenario) != -1) {
-                                        buildOpts = enableCorefxTesting ? 'skiptests' : ''
+                                        buildOpts += enableCorefxTesting ? 'skiptests' : ''
```

Comment:
Is this going to get the spacing correct? It looks like this will end up with buildOpts = 'altjitcrossgenskiptests' for x86 ryujit